src/css/AddForm.css

```diff
@@ -0,0 +1,98 @@
+.form-container {
+  border-radius: calc(0.1 * 100px);
+  font-size: 1rem;
+  display: flex;
+  justify-content: space-between;
+  width: 90rem;
+}
+
+.form {
+  /* margin: 2.313rem 6.25rem 0; */
+  width: 90rem;
+  margin: 2.313rem 4.25rem 0;
+}
+
+.input-author {
+  width: 38.5rem;
+  height: 2.813rem;
+  margin: 1.188rem 2.125rem 0 0;
+  padding: 0.813rem 30.688rem 0.813rem 1.063rem;
+  border-radius: 4px;
+  border: solid 1px var(--white);
+  background-color: var(--white-two);
+  font-size: 1rem;
+}
+
+.input-title {
+  width: 30.5rem;
+  height: 2.813rem;
+  margin: 1.188rem 2.125rem 0 0;
+  padding: 0.813rem 20.688rem 0.813rem 1.063rem;
+  border-radius: 4px;
+  border: solid 1px var(--white);
+  background-color: var(--white-two);
+  font-size: 1rem;
```

Comment:
- [ ] I see that you repeated these properties for multiple elements. It is best practice to avoid repetition of CSS properties, you can use a single class to style multiple elements instead of repeating properties for different classes.